### ISSUE TITLE
Protected Audience Input : PAS + PA

### ISSUE THREAD

**Issue by pstorm** [WICG/protected-auction-services-discussion#50]
Android explainer mentions about combining PAS and PA data in one request such that PAS undergoes real-time app-install ad selection using an inference over PAS signals while PA constraints on offline selected ads only.

Can we additionally use real time ad selection (via prepareDataForAdRetrieval & Ad-Retrieval service) for PA (protected audience), in addition to offline pre-selected (& daily rate refreshed) ads? Or, is the flow run time ad selection workflow is specifically enabled for PAS, and will be gated by infrastructure/platform for Protected Audience?

**Comment by salmanmlk** [comment]
>  Or, is the flow run time ad selection workflow is specifically enabled for PAS, and will be gated by infrastructure/platform for Protected Audience?

Yes, that flow is limited to Protected App Signals and not applicable to Protected Audience. Protected audience will continue to work the same way as it used to with pre-selected and daily refreshed ads.

**Comment by pstorm** [comment]
I was wondering that app-install-ads could be of great help in desktop world too, as gaming on desktop's continues to rise.

**Comment by galarragas** [comment]
+1 to @salmanmlk's comment, as of today, PA can only use the ads that are already stored on the device. 
To answer @pstorm: you are correct, PAS won't be available for desktop, it is an Android only feature
